src/extension.ts

```diff
@@ -946,19 +944,14 @@ function getPorts() {
 
 function describeKubernetes(explorerNode? : explorer.ResourceNode) {
     if (explorerNode) {
-        describeKubernetesCore(explorerNode.resourceId);
+        kubectl.invokeInTerminal(`describe ${explorerNode.resourceId}`);
```

Comment:
This will no longer display a separate progress indicator - I think that's okay but just checking that this is your intent.

Reply:
yes, this is by purpose. In terminal, user could get the current status of the command by himself, so no need to show an additional progress indicator.